gnosis/safe/tests/test_multi_send.py

```diff
@@ -1,17 +1,39 @@
 import logging
+from unittest import mock
 
 from django.test import TestCase
 
 from eth_account import Account
 from hexbytes import HexBytes
 
+from gnosis.eth import EthereumClient
+
 from ..multi_send import MultiSend, MultiSendOperation, MultiSendTx
 from .safe_test_case import SafeTestCaseMixin
 
 logger = logging.getLogger(__name__)
 
 
 class TestMultiSend(SafeTestCaseMixin, TestCase):
+    def test_multisend_init(self):
+        # Should try to detect the contract address
+        with self.assertRaisesMessage(
+            ValueError, "Cannot find a MultiSend contract for chainId=1337"
+        ):
+            MultiSend(self.ethereum_client)
+
+        with mock.patch.object(EthereumClient, "is_contract", return_value=True):
+            multisend = MultiSend(self.ethereum_client)
+            self.assertEqual(multisend.address, multisend.MULTISEND_ADDRESSES[0])
+
+        random_address = Account.create().address
+        multisend = MultiSend(self.ethereum_client, address=random_address)
+        self.assertEqual(multisend.address, random_address)
+
+        # Check with no ethereum_client
+        multisend = MultiSend()
```

Comment:
With no ethereum_client and without address. 
Maybe could add a test with ethereum_client without address? :thinking:

Reply:
Take a look at line `26`

Reply:
https://github.com/safe-global/safe-eth-py/pull/299/files/a4aa66392d769200dcd9398bb9fc005370b812b2#diff-475668e4347136854bf9d221fe10c3d534b6d19fa4b4980f2b074ffea13f3964R23